ocpmodels/models/forcenet.py

```diff
@@ -0,0 +1,515 @@
+import os
```

Comment:
Did we want to add the FB license header to this?

Reply:
Added